view/type/TypePage.kt

```diff
@@ -234,22 +225,23 @@ sealed class TypePage(
     @Composable
     private fun LabelSection() {
         SectionRow {
-            Form.TextBox(text = TypeDisplayName(type), leadingIcon = typeIcon(type))
+            Form.TextBox(text = ConceptSummaryText(type.conceptType), leadingIcon = conceptIcon(type.conceptType))
             EditButton { } // TODO
             Spacer(Modifier.weight(1f))
         }
     }
 
     @Composable
     private fun SupertypeSection() {
+        val supertype = type.supertype ?: type
         SectionRow {
             Form.Text(value = Label.SUPERTYPE)
             Spacer(Modifier.weight(1f))
             Form.TextButton(
-                text = type.supertype?.let { TypeDisplayName(it) } ?: AnnotatedString("(${Label.THING.lowercase()})"),
-                leadingIcon = type.supertype?.let { typeIcon(it) },
+                text = ConceptSummaryText(supertype.conceptType),
```

Comment:
We used an `AnnotatedString` method here, which could be deleted if it wasn't needed here. So in order to delete the method, we say upfront that if this type doesn't have a supertype, we'll preview the type itself here instead of its supertype.